popup.tsx

```diff
@@ -21,11 +34,15 @@ function IndexPopup() {
             authorizeButton.click()
             setTimeout(() => {
               window.close()
-            }, 2000)
```

Comment:
wish I remembered why I did this. 
I think there was an issue with 1 of the browsers and this was the quick fix.

Reply:
🤔 hmm I only tested on Chrome (wasn't even aware that this built properly in other browsers 😅), I'll go double-check